samples/python/kalman.py

```diff
@@ -86,7 +86,7 @@ def draw_cross(center, color, d):
 
             cv2.imshow("Kalman", img)
 
-            code = cv2.waitKey(100)
+            code = cv2.waitKey(100) & 0xFF
```

Comment:
I think inlining here doesn't affect performance at all since the function will wait for user interaction. So I agree it would be better to choose cleaner representation suggested by @berak and extend Python functionality as well.